dex/src/main/scala/com/wavesplatform/dex/api/websockets/package.scala

```diff
@@ -16,4 +17,14 @@ package object websockets {
     },
     tjs = Writes(x => JsString(x.toString))
   )
+
+  val assetPairKeyAsStringFormat: Format[AssetPair] = Format(
+    fjs = Reads {
+      case JsString(x) => AssetPair.extractAssetPair(x).fold(e => JsError(e.getMessage), JsSuccess(_))
+      case x           => JsError(JsPath, s"Expected a string, but got ${x.toString().take(10)}...")
+    },
+    tjs = Writes { x =>
+      JsString(x.key)
+    }
+  )
```

Comment:
We have our own domain AssetPair, maybe its a better place for this format?

Reply:
Done